code/game/atoms_movable.dm

```diff
@@ -334,7 +330,8 @@
 		reset_perspective(destination)
 		if(hud_used && length(client.parallax_layers))
 			hud_used.update_parallax()
-	update_canmove() //if the mob was asleep inside a container and then got forceMoved out we need to make them fall.
+	if(isturf(destination) && !(mobility_flags & MOBILITY_STAND))
+		fall()
```

Comment:
Does it perhaps depend on where they are for this forcemove call? I'm not sure how it would handle things if you were say, in a locker when you lost your stand mobility flag? You'd want to make sure they fall upon being pulled out, or whatever.